gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```

Comment:
@devkhan This issue is fixed. You can [look](http://jira.xwiki.org/browse/ANDAUTH-10?jql=project%20%3D%20ANDAUTH%20AND%20status%20%3D%20Open) here for more open issues for contribution.